test/unit/services/edition_publisher_test.rb

```diff
@@ -111,15 +111,6 @@ class EditionPublisherTest < ActiveSupport::TestCase
     assert publisher.perform!
   end
 
-  test "#perform! does not notify if publishing is unsuccessful" do
-    edition  = build(:imported_edition)
```

Comment:
Same here. Would this test still be useful to keep?

Could we change `:imported_edition` to e.g. `:draft_edition`?